src/Common/tests/int_hashes_perf.cpp

```diff
@@ -279,7 +279,9 @@ int main(int argc, char ** argv)
 
     if (!method || method == 1) test<identity>  (n, data.data(), "0: identity");
     if (!method || method == 2) test<intHash32> (n, data.data(), "1: intHash32");
+#if !defined(__APPLE__) /// The difference in size_t: unsigned long on Linux, unsigned long long on Mac OS.
```

Comment:
why can't revert the removing _intHash64?

Reply:
It's also possible, but the presence of this function is surprising to the reader.

Reply:
Now you cannot run one special case of one miscellaneous benchmarking tool on Mac OS... I think it's passable.

Reply:
This benchmarking tool initially served for fine tuning of the algorithms... but now it remains just for the purpose of demonstration and experiments.

Reply:
BTW, intHash64 is exactly the same as `murmurMix`.

Reply:
> BTW, intHash64 is exactly the same as `murmurMix`.

If so,  I think it's better to remove this benchmarking for all platform for consistency.